### PR TITLE
Allow setting the option for emitting JVM signatures.

### PR DESCRIPTION
**Pull request by johnedmonds**
Since it was private, it wasn't exposed in some binaries.

### CODE REVIEW

File: kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
 public class JavaIndexerConfig extends IndexerConfig {
 
   @Parameter(names = "--emit_jvm_signatures", description = "Generate vnames with jvm signatures.")
-  private boolean emitJvmSignatures;
+  protected boolean emitJvmSignatures;
```
Comment (schroederc) [inline kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java]: I would prefer adding a setter below to match the rest of the parameters.

Comment (johnedmonds) [reply kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java]: Done.

**Comment by schroederc** [comment]
Looks good other that the CLA and adding yourself to the AUTHORS file.  Thanks for the PR!

**Comment by johnedmonds** [comment]
Thanks for the review! I added my organization (Spotify) to the AUTHORS file instead of myself. Was that correct?

**Comment by schroederc** [comment]
Yeah, everything checks out!  Thanks again!

**Pull request merged**